.github/workflows/tests.yml

```diff
@@ -0,0 +1,64 @@
+name: PHPUnit
+
+on:
+  push:
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  tests:
+    name: Tests
+    runs-on: Ubuntu-20.04
+
+    strategy:
+      matrix:
+        include:
+          - php: '5.4'
+          - php: '5.5'
+          - php: '5.6'
+          - php: '7.0'
+          - php: '7.1'
+          - php: '7.2'
+          - php: '7.3'
+          - php: '7.4'
+          - php: '8.0'
+          - php: '8.1'
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          coverage: "xdebug"
+          ini-values: date.timezone=Europe/Paris,memory_limit=-1,default_socket_timeout=10,session.gc_probability=0,apc.enable_cli=1,zend.assertions=1
+          php-version: "${{ matrix.php }}"
+          extensions: "${{ env.extensions }}"
+          tools: composer
+
+      - name: Configure for PHP >= 7.1
+        if: "${{ matrix.php >= '7.1' }}"
+        run: |
+          composer require --no-update --dev symfony/error-handler "^4.4 || ^5.0"
+
+      - name: Install dependencies
+        run: |
+          composer install
+
+      - name: Run tests
+        run: |
+          vendor/bin/phpunit -v --coverage-clover=coverage.clover
+
+      - name: Code coverage
+        if: "${{ matrix.php < '8.0' }}"
+        run: |
+          wget https://scrutinizer-ci.com/ocular.phar
+          php ocular.phar code-coverage:upload --format=php-clover coverage.clover --repository=g/minkphp/Mink
```

Comment:
I suggest removing this upload of the code coverage to scrutinizer. We don't actually follow it there, and for the static analysis, psalm and phpstan have become better alternatives IMO, so I'm considering disabling scrutinizer.

Note that in that case, we should of course stop collecting the code covering during tests too

Reply:
Removed as requested